src/operator/dropout-inl.h

```diff
@@ -35,13 +35,14 @@ enum DropoutOpForwardResource {kRandom};
 namespace mxnet {
 namespace op {
 
-#if defined(USE_STATIC_MKL) && defined(_OPENMP)
+#if defined(USE_MKL) && defined(_OPENMP)
+const int nthr = omp_get_max_threads();
+std::vector<unsigned int> bernoulli_seed(nthr, 1);
```

Comment:
@piiswrong I firstly used rand() function without seed input value. But, it failed at the lint test. I am looking for another function that can pass the check and has no input. Thanks.